lhotse/recipes/timit.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+# Copyright 2021 Xiaomi Corporation (Author: Mingshuang Luo)
+# Apache 2.0
+
+from collections import defaultdict
+
+import os
+import glob
+import json
+import logging
+import shutil
+import tarfile
+import string
+from pathlib import Path
+from tqdm.auto import tqdm
+from typing import Dict, Optional, Union
+from concurrent.futures.thread import ThreadPoolExecutor
+
+from lhotse import validate_recordings_and_supervisions
+from lhotse.audio import Recording, RecordingSet
+from lhotse.supervision import SupervisionSegment, SupervisionSet
+from lhotse.utils import Pathlike, urlretrieve_progress
+
+
+def download_and_unzip(
+        target_dir: Pathlike = '.',
+        force_download: Optional[bool] = False,
+        base_url: Optional[str] = 'https://data.deepai.org/timit.zip') -> None:
+    """
+    Download and unzip the dataset, supporting both TIMIT
```

Comment:
What does `supporting both TIMIT` mean? Is there something missing?

Reply:
Em, yes, I am rewriting.